tests/unit/ReportUtilsTest.ts

```diff
@@ -4058,6 +3994,258 @@ describe('ReportUtils', () => {
         });
     });
 
+    describe('canLeaveChat', () => {
+        beforeEach(async () => {
+            jest.clearAllMocks();
+
+            await Onyx.clear();
+        });
+
+        it('should return true for root group chat', () => {
+            const report: Report = {
+                ...createRandomReport(1),
+                chatType: CONST.REPORT.CHAT_TYPE.GROUP,
+            };
+
+            expect(canLeaveChat(report, undefined, undefined)).toBe(true);
```

Comment:
Isn't this the same thing?
```suggestion
            expect(canLeaveChat(report)).toBe(true);
```